docker/mysql-build/README.md

```diff
@@ -18,12 +19,12 @@ files needed to create that image.
 
 The OpenMRS image used in [openmrs-compose.yaml](../openmrs-compose.yaml) and
 [mysql-init-compose.yaml](./mysql-init-compose.yaml),
-`us-central1-docker.pkg.dev/cloud-build-fhir/openmrs/openmrs-reference-application-distro:latest`,
-is built using the [Dockerfile](../Dockerfile) under the `docker` directory. If
-you want to use a local image, run
+`us-central1-docker.pkg.dev/cloud-build-fhir/fhir-analytics/openmrs-reference-application-extended:latest`,
+is built using [Dockerfile_openmrs_extended](../Dockerfile_openmrs_extended)
+under the `docker` directory. If you want to use a local image, run
 
 ```bash
-docker build -t openmrs-reference-application-distro:latest ../.
+docker build -t fhir-analytics/openmrs-reference-application-extended:latest ../.
```

Comment:
Don't we need the `-f` option because the Dockerfile is called `Dockerfile_openmrs_extended`?